packages/automerge-repo-network-broadcastchannel/test/index.test.ts

```diff
@@ -36,9 +36,9 @@ describe("BroadcastChannel", () => {
       }, 100)
     })
 
-    a.connect("a" as PeerId)
-    b.connect("b" as PeerId)
-    c.connect("c" as PeerId)
+    a.connect("a" as PeerId, {})
```

Comment:
Is this actually a backwards incompatible change that we made to `NetworkAdapter`? If so it seems to me like we should instead add a default argument for the peer metadata.

Reply:
That sounds like a more sensible path to me. I shall make that change instead

Reply:
Updated the adapters to make peerMetadata optional